File: 学校上课写的/7.计导的斐波那契/Fibonacci_2.c
#include <stdio.h>
#include <time.h>

int main(){
    long long a = 1,b = 0,c;
    int term, start, end;
    printf("请问你想看到斐波那契数列的第几项:");
    scanf("%d", &term);
    start = clock();
    for(int i = 1;i <= term;i++){
        printf("F[%d] = %lld\n", i, a);
        c = b;
        b = a;
        a += c;
    }
    end = clock();
    printf("耗时: %dms\n", end - start);
    return 0;
}

File: 学校上课写的/7.计导的斐波那契/fibonacci_2.py
import time

if __name__ == "__main__":
    a, b = 1, 0
    term = int(input("请问你想看到斐波那契数列的第几项: "))
    start = time.time()
    for i in range(1, term + 1):
        print(f"F[{i}] = {a}")
        b, c = a, b
        a += c
    end = time.time()
    print(f"耗时: {(end - start):.10f}ms\n")
